velox/type/tz/TimeZoneMap.cpp

```diff
@@ -16,12 +16,12 @@
 
 #include "velox/type/tz/TimeZoneMap.h"
 #include <fmt/core.h>
-#include <unordered_map>
+#include <folly/container/F14Map.h>
 
 namespace facebook::velox::util {
 
 // Defined on TimeZoneDatabase.cpp
-extern const std::unordered_map<int64_t, std::string>& getTimeZoneDB();
+extern const folly::F14FastMap<int64_t, std::string>& getTimeZoneDB();
```

Comment:
I think the problem is that you changed the return type of getTimeZoneDB() function which is defined in TimeZoneDatabase.cpp. Revert this change to see if it makes the tests pass.